content/posts/2021/release-0.8.0.md

```diff
@@ -1,53 +1,315 @@
 ---
 title: "REANA 0.8.0 is released"
-date: 2021-11-29T10:19:06+01:00
+date: 2021-11-30T10:19:06+01:00
 tags:
   - release
-  - client
-  - cluster
```

Comment:
In other releases we have `cluster` and `client`.

Reply:
Added back. My worry was that we might have client/cluster almost everywhere in each post, which would loose its narrowing meaning, as it were, due to not offering enough granularity... But I guess we can revisit the tagging and the tag cloud later down the line.